src/Elliptic/Systems/Xcts/Geometry.hpp

```diff
@@ -0,0 +1,18 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+namespace Xcts {
+/// \brief Types of conformal geometries for the XCTS equations
+enum class Geometry {
```

Comment:
I don't understand why we need separate, identical tags in Poisson and Xcts.  Wouldn't code re-use point toward just having one Geometry tag?   And also, are separate Poisson::Geometry and Xcts::Geometry tags going to cause inconsistencies later on?  E.g. what if Geometry-templated XCTS code wants to re-use Poisson-code?

Reply:
The enums could be merged, though I didn't do that so far because:

1. I don't think the code re-use is super important here since it's just an enum to get human-readable labels and there's no logic involved. Generalizing and re-using code is nice, but taken too far it can make the code more difficult to follow.
2. I'm not sure where to place it in the codebase since `elliptic::Geometry` doesn't seem right; it could be more general than the elliptic code, but such generality is probably unnecessary.
3. I don't think inconsistencies with Poisson code will be an issue. The Poisson (and elasticity) code is called internally to re-use implementations, but none of that is exposed to the public interface. Also the Poisson and elasticity functions are called directly (e.g. `Poisson::curved_fluxes(...)`), not using the `Poisson::Geometry` enum. The enum is there just to have nicer labels for the template parameters.

Reply:
I am fine if you're fine.